modules/c1xBidAdapter.js

```diff
@@ -0,0 +1,207 @@
+      response: 'json',
+      compress: 'gzip'
+    }
+
+    // for GDPR support
+    if (bidderRequest && bidderRequest.gdprConsent) {
+      payload['consent_string'] = bidderRequest.gdprConsent.consentString;
+      payload['consent_required'] = (typeof bidderRequest.gdprConsent.gdprApplies === 'boolean') ? bidderRequest.gdprConsent.gdprApplies.toString() : 'true'
+      ;
+    }
+
+    // if (pixelId) {
+    //   pixelUrl = (useSSL ? 'https:' : 'http:') + PIXEL_ENDPOINT + pixelId;
+    //   if (payload.consent_required) {
+    //     pixelUrl += '&gdpr=' + (bidderRequest.gdprConsent.gdprApplies ? 1 : 0);
+    //     pixelUrl += '&consent=' + encodeURIComponent(bidderRequest.gdprConsent.consentString || '');
+    //   }
+    //   userSync.registerSync('image', BIDDER_CODE, pixelUrl);
+    // }
+
+    // tagObj['site'] = _bidderSettings.c1x.siteId | '';
+
+    Object.assign(payload, tagObj);
+    let payloadString = stringifyPayload(payload);
+    // ServerRequest object
+    bidRequest.push({
+      method: 'GET',
+      url: URL,
+      data: payloadString,
+      bids: bidIdTags
+    })
+    return bidRequest;
+  },
+
+  interpretResponse: function(serverResponse, requests) {
+    serverResponse = serverResponse.body;
+    requests = requests.bids || [];
+    const currency = 'USD';
+    const bidResponses = [];
+    let netRevenue = false;
+
+    if (!serverResponse || serverResponse.error) {
+      let errorMessage = serverResponse.error;
+      logError(LOG_MSG.invalidBid + errorMessage);
+      return bidResponses;
+    } else {
+      serverResponse.forEach(bid => {
+        logInfo(bid)
+        if (bid.bid) {
+          if (bid.bidType === 'NET_BID') {
+            netRevenue = !netRevenue;
+          }
+          const curBid = {
+            requestId: bid.bidId,
+            width: bid.width,
+            height: bid.height,
+            cpm: bid.cpm,
+            ad: bid.ad,
+            creativeId: bid.crid,
+            currency: currency,
+            ttl: 300,
+            netRevenue: netRevenue
+          };
+
+          if (bid.dealId) {
+            curBid['dealId'] = bid.dealId
+          }
+
+          for (let i = 0; i < requests.length; i++) {
+            if (bid.adId === requests[i].adUnitCode) {
+              curBid.requestId = requests[i].bidId;
+            }
+          }
+          logInfo(LOG_MSG.bidWin + bid.adId + ' size: ' + curBid.width + 'x' + curBid.height);
+          bidResponses.push(curBid);
+        } else {
+          // no bid
+          logInfo(LOG_MSG.noBid + bid.adId);
+        }
+      });
+    }
+
+    return bidResponses;
+  }
+
+}
+
+function bidToTag(bid, index) {
+  const tag = {};
+  const adIndex = 'a' + (index + 1).toString(); // ad unit id for c1x
+  const sizeKey = adIndex + 's';
+  const priceKey = adIndex + 'p';
+  const dealKey = adIndex + 'd';
+  // TODO: Multiple Floor Prices
+
+  const sizesArr = bid.sizes;
+  const floorPriceMap = bid.params.floorPriceMap || '';
```

Comment:
Please update to check this to check `bid.getBidFloor` if the params for `floorPriceMap` are not defined.

Reply:
this change is done and added to the existing PR cc @vishnu1211